.ci/RELEASE_NOTES_GUIDE.md

```diff
@@ -117,12 +117,33 @@ Starting on Nov 1, 2019, Cloud Functions API will be private by default. Add app
 
 Don’t write notes like:
 - Add compute_instance resource
```

Comment:
My preference is more examples that are close but show 1 way that they are incorrect. It's harder to parse comment that is wrong in many ways and use that to construct something better.